### ISSUE TITLE
Use an alternative image to Crunchy Containers?

### ISSUE THREAD

**Issue by twalk4821** [CrunchyData/postgres-operator#145]
Hi -

After struggling with this most of yesterday, I finally managed to deploy and expose the operator service to a google cloud k8s cluster. However, now I am stuck with a new problem, which is using another image for the postgres db, such as the timescaledb image. Is this capability supported? 
Thanks -
Tyler

**Comment by twalk4821** [comment]
Also having another issue -- I can't seem to psql into the instance: keeps telling me password authentication failed no matter what username/password combination I try.

**Comment by twalk4821** [comment]
Answered my second question: that the passwords are encoded prior to being stored as a k8 secret

**Comment by jmccormick2001** [comment]
good to hear...regarding the timescaledb container, right now the postgres-operator just supports the crunchy postgres container, mostly due to the various hooks in the container that we take advantage of and also some of the conventions....someone else asked about adding timescale extension into the crunchy-postgres container, that might be possible, worth looking into.

**Comment by twalk4821** [comment]
Thanks for getting back to me. Yeah I guess it makes sense that this wouldn't be supported out of the box, since the postgres container is part of an ecosystem of crunchy containers that need to be compatible. I think what we are going to do is try to install all the necessary extensions on top of the crunchy container. I think that should work.

**Comment by jmccormick2001** [comment]
yep, I think that's the right approach, add the timescaledb extension into the crunchy container and do a custom build of that, you can modify the setup.sql to create those extensions for you when the container starts and inits.

**Comment by k1ng440** [comment]
were you available to install the timescale container?

**Comment by jmccormick2001** [comment]
I think the position right now is for people needing the timescaledb extension they can just take our normal Postgres container, add the extension, and create their own container image to use.  We might support the timescaledb extension in the future if enough interest arises for us to support one.
